README.md

```diff
@@ -1,36 +1,156 @@
+      </li>
+      </ul>
+    </td>
+  </tr>
+
+  <tr>
+    <td><a href="https://docs.wavefront.com/wavefront_sdks.html#sdks-that-instrument-frameworks">Framework SDK</a></td>
+    <td align="justify">Reports predefined traces, metrics, and histograms from the APIs of a supported app framework. Lets you get started quickly with minimal code changes.</td>
+    <td>
+      <ul>
+      <li><b>Java</b>:
+      <a href="https://github.com/wavefrontHQ/wavefront-dropwizard-sdk-java">Dropwizard</a> <b>|</b> <a href="https://github.com/wavefrontHQ/wavefront-gRPC-sdk-java">gRPC</a> <b>|</b> <a href="https://github.com/wavefrontHQ/wavefront-jaxrs-sdk-java">JAX-RS</a> <b>|</b> <a href="https://github.com/wavefrontHQ/wavefront-jersey-sdk-java">Jersey</a></li>
+      <li><b>.Net/C#</b>: 
+      <a href="https://github.com/wavefrontHQ/wavefront-aspnetcore-sdk-csharp">ASP.Net core</a> </li>
+      <!--- [Python](wavefront_sdks_python.html#python-sdks-that-instrument-frameworks) --->
+      </ul>
+    </td>
+  </tr>
+
+  <tr>
+    <td><a href="https://docs.wavefront.com/wavefront_sdks.html#sdks-for-sending-raw-data-to-wavefront">Sender SDK</a></td>
+    <td align="justify">Lets you send raw values to Wavefront for storage as metrics, histograms, or traces, e.g., to import CSV data into Wavefront. 
+    </td>
+    <td>
+      <ul>
+      <li>
+      <b>Java</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-java">Sender SDK</a>
+      </li>
+      <li>
+      <b>Python</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-python">Sender SDK</a>
+      </li>
+      <li>
+      <b>Go</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-go">Sender SDK</a>
+      </li>
+      <li>
+      <b>.Net/C#</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-csharp">Sender SDK</a>
+      </li>
+      <li>
+      <b>C++</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-cpp">Sender SDK</a>
+      </li>
+      </ul>
+    </td>
+  </tr>
+</table> 
+
+## Requirements and Installation
+
+* **Supported Frameworks**
   * .NET Framework (>= 4.5.2)
   * .NET Standard (>= 2.0)
 
-## Installation
-Install the [NuGet package](https://www.nuget.org/packages/Wavefront.OpenTracing.SDK.CSharp/).
+* **Installation** <br/>
+  Install the [NuGet package](https://www.nuget.org/packages/Wavefront.OpenTracing.SDK.CSharp/).
+  * Package Manager Console
+    ```
+    PM> Install-Package Wavefront.OpenTracing.SDK.CSharp
+    ```
+  * .NET CLI Console
+    ```
+    > dotnet add package Wavefront.OpenTracing.SDK.CSharp
+    ```
 
-### Package Manager Console
+## Usage
 
-```
-PM> Install-Package Wavefront.OpenTracing.SDK.CSharp
-```
+[Tracer](https://github.com/opentracing/specification/blob/master/specification.md#tracer) is an OpenTracing [interface](https://github.com/opentracing/opentracing-csharp#initialization) for creating spans and propagating them across arbitrary transports.
 
-### .NET CLI Console
+This SDK provides a `WavefrontTracer` to:
```

Comment:
This SDK provides a `WavefrontTracer` that:
(or some other way to make the sentence flow. could also change the bullet text).